README.md

```diff
@@ -26,3 +26,12 @@ A custom scheduler to easily run tasks either in the back- or the foreground.
 
 * [UIView+Autolayout](https://github.com/novoda/ios-demos/blob/master/UIView%2BAutolayout.swift)
 extends UIView; Import this into your project to be able to use constraints programmatically in a straightforward way.
+
+## How to add a new project
+ - Pull from master and create a new branch with your project and/or changes
+ - If you are creating a new project make sure that XCode did not created a `.git` file inside of your folder. You can check that by going inside the project folder on terminal and entering `ls -la` if there is a `.git` file it will conflict with the git repository of the `ios-demos` repo. Delete it by entering `rm -rf .git` while being inside your project folder.
+- Push your branch and make a PR. Your PR should have:
+  1. Description of what the project is/ what changes are you making and why
+  2. Screenshots if is a new project so it can be reviewed properly
+
+You will need to wait for someone to approved and merge your PR
```

Comment:
```
Someone will code review your PR as soon as possible. From there, it could be approved right away
and merged — great job! :tada: — or it may require some more work. Don't worry if that's the case,
the team will work with you to make the necessary changes and get your PR merged!
```